lib/trie/mem_test.go

```diff
@@ -16,7 +16,7 @@ func Test_Trie_MemoryUsage(t *testing.T) {
 	// Set skip to false to run the test.
 	// This test should be run on its own since it interacts
 	// with the Go garbage collector.
-	const skip = true
+	const skip = false
```

Comment:
Will this effect effect tests if this test isn't run in isolation? Looking at above comment

Reply:
Good point, changed it back to `true` since it should be run isolated. The CI takes care of changing that constant (ugly `sed` hack) to `true` when needed.